### DISCUSSION TITLE
Scroll=ScrollMode.ALWAYS horizontal sliding is very laggy

### DISCUSSION THREAD

**Discussion by JFScorpio** [flet-dev/flet#4085] [Q&A]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the bug

Scroll=ScrollMode.ALWAYS / AUTO  horizontal sliding is very laggy   in Row

### Code sample

Container(

    content=Row(
        scroll=ScrollMode.ADAPTIVE,
        spacing=30,
        controls=[
            Container(
                height=260,
                width=260,
                bgcolor=tabs_color,
                border_radius=br,
                content=Row(
                    spacing=15,
                    alignment='center',
                    controls=[
                        Container(
                            height=70,
                            width=70,
                            border_radius=15,
                            bgcolor=tabs_grey,
                            content=Image(
                                src='plus.png',
                                scale=0.5
                            )
                        ),
                        Text(
                            value='Add URL',
                            font_family='montserrat medium',
                            size=20,
                        )
                    ]
                )

            ),
            ]*10
    )
)

### To reproduce

see demo code

### Screenshots / Videos

<details open>
<summary>Captures</summary>

[Upload media here]

</details>

### Operating System

Windows

### Operating system details

windows 11

### Flet version

0.24.0 and 0.24.1

### Regression

Yes, it used to work in a previous Flet version (please specify the version in additional details)

### Logs

<details open><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

**Comment by tsiens** [comment]
me too,
Very awkward in the row, very smooth in the column
